clients/da-vinci-client/src/main/java/com/linkedin/davinci/stats/RocksDBMemoryStats.java

```diff
@@ -103,6 +107,27 @@ public RocksDBMemoryStats(MetricsRepository metricsRepository, String name, bool
         }
         return total;
       }, metric));
+      registerSensor(new AsyncGauge((ignored, ignored2) -> {
+        Long total = 0L;
+        // Lock down the list of RocksDB interfaces while the collection is ongoing
+        synchronized (hostedRocksDBPartitions) {
+          for (RocksDBStoragePartition dbPartition: hostedRocksDBPartitions.values()) {
+            if (dbPartition instanceof ReplicationMetadataRocksDBStoragePartition) {
+              try {
+                total += dbPartition.getRocksDBCFStatValue(metric);
+              } catch (VeniceException e) {
+                LOGGER.warn("Could not get rocksDB CF metric {} with error:", metric, e);
+                continue;
+              }
+            }
+            if (INSTANCE_METRIC_DOMAINS.contains(metric)) {
```

Comment:
Shall we check whether separate block cache is enabled or not before emitting another set of block cache metrics for `rmd` CF?